src/views/ingresses/list/utils/utils.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
This file is a util file but return element, please extract the element to a different component file and change the util.tsx to util.ts

Reply:
I moved the element to a different component, but can't change the extension type since the function is returning an element.

Reply:
if u return Component and not <Component/> it will work

Reply:
Or, even better, getHosts will return hostStr || null, and consumer will handle the present of JSX, separating logic from presentation.

Reply:
Done!